emubd/lfs_emubd.c

```diff
@@ -47,19 +47,33 @@ int lfs_emubd_create(const struct lfs_config *cfg, const char *path) {
 
     // Load stats to continue incrementing
     snprintf(emu->child, LFS_NAME_MAX, "stats");
+
     FILE *f = fopen(emu->path, "r");
-    if (!f) {
+    if (!f && errno != ENOENT) {
         return -errno;
     }
-
-    size_t res = fread(&emu->stats, sizeof(emu->stats), 1, f);
-    if (res < 1) {
-        return -errno;
+    else if (errno == ENOENT) {
```

Comment:
I know I'm being nitpicky, but it helps to have a consistent style. Could you adopt either:
``` c
if (blah) {
    return 0;
}

if (blah) {
    blah;
}
```

or

``` c
if (blah) {
    return 0;
} else if (blah) {
    blah;
}
```

I can make this change when squashing the PR if you don't.